example_configs/wikijs.md

```diff
@@ -0,0 +1,64 @@
+# Configuration for WikiJS
+Replace `dc=example,dc=com` with your LLDAP configured domain.
+### LDAP URL
+```
+ldap://lldap:3890
+```
+### Admin Bind DN
+```
+cn=admin,ou=people,dc=example,dc=com
```

Comment:
Small nit : it's better to use uid here instead of cn

Reply:
OK, will change that.